### ISSUE TITLE
Release version for elasticsearch 2.4.1

### ISSUE THREAD

**Issue by szydan** [sirensolutions/siren-join#96]
We would need it in near future for kibi 4.6

**Comment by lmangani** [comment]
PR coming

**Comment by GaneshbabuRamamoorthy** [comment]
Hi All,

Do we have an update of latest version SIREn Join for Elasticsearch 2.4.0?

Is Siren join plugin version 2.3.5 will work for ES 2.4.0?

Please kindly let us know and it would be very helpful for our team to develop ES query using co-ordinate search.

Thanks,
Ganeshbabu R

**Comment by GaneshbabuRamamoorthy** [comment]
Hi @rendel 

I tried by installing siren-join version 2.3.5 with Elasticsearch 2.4.0 and I am getting incompatible error msg.

Below are the error msg,
```
[esadmin@mue2rhegrdd001 elasticsearch-2.4.0]$ bin/plugin install solutions.siren/siren-join/2.3.5
-> Installing solutions.siren/siren-join/2.3.5...
Trying https://download.elastic.co/solutions.siren/siren-join/siren-join-2.3.5.zip ...
Trying https://search.maven.org/remotecontent?filepath=solutions/siren/siren-join/2.3.5/siren-join-2.3.5.zip ...
Downloading .........................................DONE
Verifying https://search.maven.org/remotecontent?filepath=solutions/siren/siren-join/2.3.5/siren-join-2.3.5.zip checksums if available ...
Downloading .DONE
ERROR: Plugin [siren-join] is incompatible with Elasticsearch [2.4.0]. Was designed for version [2.3.5]
```
Please let me know is there any chance of latest version available for ES 2.4.0

Thanks,
Ganeshbabu R

**Comment by rendel** [comment]
2.4.1 is released and pushed to maven repository
